### ISSUE TITLE
[Customer Query] Use array in script localization

### ISSUE THREAD

**Issue by SID177** [rtMediaWP/rtMedia#1573]
If we directly add any text like this
```
wp_localize_script( 'script-handle', 'objectName', esc_html__( "Something's fishy", 'text-domain' ) );
```
Then `Something's fishy` will be converted into `Something&#039;s fishy`.

Instead if we add this in array which is proper format of [wp_enqueue_script](https://developer.wordpress.org/reference/functions/wp_localize_script/), it won't convert `'` into `&#039;`.
```
wp_localize_script( 'script-handle', 'objectName', array( 'data' => esc_html__( "Something's fishy", 'text-domain' ) ) );
```

----------------

Customer screenshot
![Screen_Shot_2020-03-10_at_4_21_24_PM](https://user-images.githubusercontent.com/23107546/76313046-347a5580-62fa-11ea-8c5b-4e85e1148afd.jpg)

**Comment by SID177** [comment]
As a quick workaround, we've provided the customer a custom code
```
if ( ! function_exists( 'rtmedia_custom_code_wp_enqueue_scripts' ) ) {
	/**
	 * Change default add more files text.
	 * 
	 * @return void
	 */
	function rtmedia_custom_code_wp_enqueue_scripts() {
		wp_localize_script( 'rtmedia-main', 'rtmedia_add_more_files_msg', __( 'Add more files', 'buddypress-media' ) );
	}
}
add_action( 'wp_enqueue_scripts', 'rtmedia_custom_code_wp_enqueue_scripts', 1000 );
```

After the fix is released, we need to notifiy the customer about it and ask her to remove this custom code
ticket - https://rtmedia.io/ticket/2568706/

**Comment by kjnanda** [comment]
@SID177 As we discussed, I did the changes in the respected file and that looks good to me.

**Comment by kjnanda** [comment]
This is working as expected and tested with `dev` branch. ✅

**Comment by kjnanda** [comment]
Closing this issue as this has been fixed and release in - V4.6.1
Also, a reply to the customer is sent.
